routing/CMakeLists.txt

```diff
@@ -183,6 +183,8 @@ set(SRC
 
 omim_add_library(${PROJECT_NAME} ${SRC})
 
+target_include_directories(${PROJECT_NAME} SYSTEM PUBLIC ${OMIM_ROOT}/3party/skarupke/include)
```

Comment:
Had to move `skarupke` lib to nested folder to be able to declare it as "system"(clang-tidy doesn't alert about problems in "system" headers)